.github/workflows/deploy-auto.yml

```
@@ -0,0 +1,21 @@
+# Automatically create a new version/tag, if new commits are available since the last tag
+name: Auto-tagger
+
+on:
+  schedule:
+    - cron: '51 15 */4 * *' # Every 4 days at 15:51 https://crontab.guru/#51_15_*/4_*_*
+
+jobs:
+  create-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-token@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          if [[ $(git tag -l --points-at HEAD) = "" ]]; then
+            export VER=$(npx daily-version)
+            git tag $VER -m $VER
+            git push origin $VER
+          fi
```

I hope someone at some point figures out a way to move this condition to the workflow itself, like: 
```suggestion
      - if: is HEAD not tagged?
        run: |
          VER=$(npx daily-version)
          git tag $VER -m $VER
          git push origin $VER
```

Using: https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions